docker-compose.yml

```diff
@@ -84,6 +98,7 @@ services:
       AWS_ACCESS_KEY_ID: ""
       AWS_SECRET_ACCESS_KEY: ""
       GIT_SSH_COMMAND: "ssh -F /home/packit/.ssh/config"
+      SQLALCHEMY_ECHO: 1
```

Comment:
The log is quite verbose with it, but it doesn't hurt in the dev environment, so :+1: